cmd/users.go

```diff
@@ -267,11 +256,22 @@ func userRoles(cmd *cobra.Command, args []string) error {
 	return stdout(roles)
 }
 
-func openJSON(name string) (*os.File, error) {
+func readJSON(name string, object interface{}) error {
 	file, err := os.Open(name)
 	if err != nil {
-		return nil, err
+		return err
 	}
+	defer file.Close()
 
-	return file, err
+	data, err := ioutil.ReadAll(file)
+	if err != nil {
+		return err
+	}
+
+	err = json.Unmarshal(data, &object)
+	if err != nil {
+		return err
+	}
+
+	return err
```

Comment:
Line 272-276 does not make much sense. Either return err in all cases or nil explicitly